src/components/MovieCard.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PosterMovie from './PosterMovie';
+import ProgressBar from './ProgressBar';
+import DescriptionMovie from './DescriptionMovie';
+
+export default function MovieCard({movieData, background}) {
+    return (
+        <div className='movie--card'>
+            <li 
+                key={movieData.id} 
```

Comment:
Distructure `movieData`.

```suggestion
const { id, score, average, title, releaseDate, posterPath } = movieData;
```